lib/streamlit/string_util.py

```diff
@@ -24,6 +24,10 @@
     from streamlit.type_util import SupportsStr
 
 
+ESCAPED_EMOJI = [re.escape(e) for e in sorted(ALL_EMOJIS, reverse=True)]
```

Comment:
Does the `sorted(ALL_EMOJIS, reverse=True)` here prevent that we are matching only a part of a combined emoji? Is the longest Unicode sequence always on top?

Reply:
Yep, let me add a comment to clarify this